crawler-cli/src/main/java/org/gbif/crawler/pipelines/dwca/DwCAToAvroConverterCallBack.java

```diff
@@ -2,20 +2,65 @@
 
 import org.gbif.common.messaging.AbstractMessageCallback;
 import org.gbif.common.messaging.api.messages.DwcaValidationFinishedMessage;
+import org.gbif.pipelines.core.io.DwCAReader;
+import org.gbif.pipelines.io.avro.ExtendedRecord;
 
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Objects;
+
+import org.apache.avro.file.DataFileWriter;
+import org.apache.avro.io.DatumWriter;
+import org.apache.avro.specific.SpecificDatumWriter;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
  * Call back which is called when the {@link org.gbif.common.messaging.api.messages.DwcaValidationFinishedMessage } is received.
- * ToDo Implement DwCAToAvro Conversion
  */
 public class DwCAToAvroConverterCallBack extends AbstractMessageCallback<DwcaValidationFinishedMessage> {
```

Comment:
can we rename it to DwCAToAvroCallBack ? (to keep consistency)

Reply:
done